job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/dto/ProfileDTO.java

```diff
@@ -0,0 +1,67 @@
+package org.verapdf.webapp.jobservice.model.dto;
+
+import org.verapdf.webapp.jobservice.model.entity.enums.Profile;
+
+import java.util.Objects;
+
+public class ProfileDTO {
+
+	private Profile profileName;
+	private String humanReadableName;
+	private boolean available;
+
+	public ProfileDTO(Profile profileName) {
```

Comment:
the constructor should has 2 arguments (Profile + boolean available). The available flag should be evaluated outside of dto, as dto should be simple

Reply:
The constructor now have 3 argument to let us define available and enabled profiles. (Profile, boolean available, boolean enabled)